debezium-testing/pom.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>debezium-testing</artifactId>
```

Comment:
What's the reasoning for adding this intermediary POM?

Reply:
In google groups you mention to add the `KafkaCluster` utility as a testing tool as well. So probably this class should go into its own module inside testing module. So I prepared for this.

Reply:
Ah, yes. Kinda thought that'd be the reason, but wasn't sure. 👍